src/connectors/tenable.py

```diff
@@ -0,0 +1,124 @@
+"""Tenable Settings
+Collects Tenable Settings using a Service User’s API Key
+"""
+from tenable.io import TenableIO
+from datetime import datetime
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+CONNECTION_OPTIONS = [
+    {
+        'type': 'select',
+        'options': [
+            {'value': 'user', 'label': "Tenable Users"},
+        ],
+        'name': 'connection_type',
+        'title': "Settings Type",
+        'prompt': "The type of Tenable Settings information you are ingesting to Snowflake.",
+        'required': True
```

Comment:
add a `'default': 'user',` should give people two less clicks to set this up, since it's the only one for now